testing/setup.sh

```diff
@@ -322,6 +326,26 @@ if ((IMAGE)); then
       IMAGE_SCRIPT="./helpers/image.sh"
     fi
 
+    AUTO_POOL_COMPAT=1
+    for environ in "${ENVIRONS[@]}"; do
+      if [[ "${environ}" =~ "POOL_COMPAT" ]]; then
```

Comment:
I'd either anchor this as `"^POOL_COMPAT="` or use a case block matching `POOL_COMPAT=*`